src/validateRegisterForm.test.js

```diff
@@ -26,5 +26,119 @@ describe(`Function 'validateRegisterForm':`, () => {
     expect(invalidPassword.message).toBe('Password is invalid.');
   });
 
-  // write more tests here
+  it(`should return error for valid email and password of length 7`,
+    () => {
+      const invalidPassword = validateRegisterForm('test@mail.com',
+        'P@sswor');
+
+      expect(invalidPassword.code).toBe(422);
+      expect(invalidPassword.message).toBe('Password is invalid.');
+    });
+
+  it(`should return error for valid email and password less 
+  than 8 characters`, () => {
+    const invalidPassword = validateRegisterForm('test@mail.com', 'P@ss1');
+
+    expect(invalidPassword.code).toBe(422);
+    expect(invalidPassword.message).toBe('Password is invalid.');
+  });
+
+  it(`should return success message for valid email
+   and password of length 16`, () => {
+    const isValid = validateRegisterForm('test@mail.com',
+      'P@ssword1234567');
+
+    expect(isValid.code).toBe(200);
+    expect(isValid.message).toBe('Email and password are valid.');
+  });
+
+  it(`should return error for valid email and password less 
+  than 17 characters`, () => {
```

Comment:
This test case is misleading as it states 'should return error for valid email and password less than 17 characters', but the password 'P@ss1' is only 5 characters long.